pkg/validate/dynamic/dynamic.go

```diff
@@ -764,23 +766,41 @@ func (dv *dynamic) ValidateSubnets(ctx context.Context, oc *api.OpenShiftCluster
 			)
 		}
 
-		_, net, err := net.ParseCIDR(*ss.AddressPrefix)
-		if err != nil {
-			return err
+		// Handle both addressPrefix & addressPrefixes
+		if ss.AddressPrefix == nil {
+			for _, address := range *ss.AddressPrefixes {
```

Comment:
Stupid question: why we don't need
```go
if ss.AddressPrefixes != nil {
```
before iterating *ss.AddressPrefixes (note the plural)?

Reply:
One or the other will always be set.